packages/component-model-user/resolvers.js

```diff
@@ -0,0 +1,12 @@
+const User = require('./index').model
```

Comment:
can this go relatively to the entity file rather than through the index? would be good to only have the .model suffix as a temp thing